catalyst/utils/torch.py

```diff
@@ -124,6 +124,27 @@ def get_optimizer_momentum(optimizer: Optimizer) -> float:
     return betas[0] if betas is not None else momentum
 
 
+def get_optimizer_momentum_list(
```

Comment:
could you please also add it 
- to `__all__` in the bottom
- to import in the `catalyst/utils/__init__.py`
?

Reply:
done